fe/fe-core/src/main/java/org/apache/doris/statistics/StatisticsCache.java

```diff
@@ -92,6 +100,24 @@ public ColumnStatistic getColumnStatistics(long catalogId, long dbId, long tblId
         return ColumnStatistic.UNKNOWN;
     }
 
+    public PartitionStatistic getPartitionStatistics(long catalogId, long dbId, long tblId, long idxId,
```

Comment:
it would be better to rename to getPartitionColumnStatistics()
And how about changing params of getColumnStatistics(..., partId, ...) and make getPartitionStatistics private

Reply:
Discussed offline, will keep is as is.